bin/triton-mysql.py

```diff
@@ -793,7 +793,7 @@ def write_snapshot(conn):
     # create_snapshot call and return. The snapshot process will be
     # re-parented to Containerpilot
     set_backup_ttl()
-    subprocess.Popen(['python', '/bin/triton-mysql.py', 'create_snapshot'])
+    subprocess.Popen(['python', '/usr/local/bin/triton-mysql.py', 'create_snapshot'])
```

Comment:
Couldn't this rely on `PATH` too, or does `python` not check `PATH` when looking for the file?  Any particular reason to `python ...` explicitly here and not just `+x` and shebang the script?